src/index.js

```diff
@@ -0,0 +1,135 @@
+const landscape = document.getElementById('landscape');
+const skySelect = document.getElementById('skySelect');
+const sky = document.getElementById('sky');
+const currentTempButton = document.getElementById('currentTempButton');
+
+
+const updateTemprature = () => {
+    // const Celsius = Math.round((temperature-32)*5/9);
+    let temp;
+    if (isCelsius){
+      tempValue.textContent = `${temperature}°C`;
+    } else {
+      temp = getTempInFahrenheit(temperature) 
+      tempValue.textContent = `${temp}°F`;
+    }
+    updateColorTemp(temperature);
+    updateWeatherGarden(temperature);
+};
+// Celsius changes color
+const updateColorTemp = (temperature) => {
+
+    tempValue.className = '';
+
+  if (temperature >= 27) {
+    tempValue.classList.add('red');
+  } else if (temperature >= 21) {
+    tempValue.classList.add('orange');
+  } else if (temperature >= 15) {
+    tempValue.classList.add('yellow');
+  } else if (temperature >= 10) {
+    tempValue.classList.add('green');
+  } else {
+    tempValue.classList.add('teal');
+  }
+};
+
+const updateWeatherGarden = (temperature) => {
+  if (temperature >= 27) {
+    landscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+  } else if (temperature >= 21) {
+    landscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+  } else if (temperature >= 15) {
+    landscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+  } else if (temperature >= 10) {
+    landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  } else {
+    landscape.textContent = "🌲🌲❄️❄️🐘❄️🌲❄️🌲";
+  }
+};
+
+const updateSky = () => {
+    if (skySelect.value === "Sunny"){
+        sky.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+    } else if (skySelect.value === "Cloudy"){
+        sky.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+    } else if (skySelect.value === "Rainy"){
+        sky.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+    } else {
+        sky.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+    }
+};
+
+const convertTemp = () => {
+  const temperatureHeader = document.getElementById('temperatureHeader');
+  if (isCelsius) { 
+    convertTempButton.textContent = "Convert to °C";
+    temperatureHeader.textContent = "Temperature (°F)";
+  } else {
+    convertTempButton.textContent = "Convert to °F";
+    temperatureHeader.textContent = "Temperature (°C)";
+  }
+  isCelsius = !isCelsius;
+  updateTemprature();
+};
+
+// Auxiliar functions: 
+const getTempInFahrenheit = (temperatureC) => Math.round((temperatureC) * 9/5 + 32);
+
+// Add here: calling APIs
+currentTempButton.addEventListener('click', () => {
+    axios.get('http://127.0.0.1:5000/location',{
+        params: {q: cityNameInput.value}
+    })
+    .then ((response) => {
+        const {lat, lon} = response.data[0];
+        return axios.get('http://127.0.0.1:5000/weather',{
+            params: {lat, lon}
+        });
+    })
+    .then ((response_weather) => {
+        // console.log(response_weather);
+        const kelvinTemperature = response_weather.data.main.temp;
+        // Celsius = Kelvin - 273.15
```

Comment:
```suggestion
        const kelvinTemperature = response_weather.data.main.temp;
```

Clean up debugging print statements and notes that you left for yourself before opening a PR so that your code doesn't become cluttered with comments.